Formula/forego.rb

```diff
@@ -16,19 +16,18 @@ class Forego < Formula
   end
 
   depends_on "go" => :build
-  depends_on "godep" => :build
 
   def install
     ENV["GOPATH"] = buildpath
-    mkdir_p buildpath/"src/github.com/ddollar/"
-    ln_sf buildpath, buildpath/"src/github.com/ddollar/forego"
-
-    ldflags = "-X main.Version=#{version} -X main.allowUpdate=false"
-    system "godep", "go", "build", "-ldflags", ldflags, "-o", bin/"forego"
+    (buildpath/"src/github.com/ddollar/forego").install buildpath.children
+    cd "src/github.com/ddollar/forego" do
+      system "go", "build", "-o", bin/"forego", "-ldflags", "-X main.Version=#{version} -X main.allowUpdate=false"
```

Comment:
Note that I've moved `"-X main.Version=#{version} -X main.allowUpdate=false"` to the next line as well to avoid going passed the eightieth column.